mandatory/3-function-output.js

```diff
@@ -1,16 +1,20 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// Returns a random integer from 0 to 9
```

Comment:
You could add further comments to explain how Math.random() returns a decimal value which is then scaled to a desired range.

Reply:
I was concentrated so much on JavaScript that I didn`t notice the message about "add comment..." ))) Thank you, Shahid and Katie!

Reply:
I was concentrated so much on JavaScript that I didn`t notice the message about "add comment..." ))) Thank you, Shahid and Katie!